### ISSUE TITLE
ORCA Configuration Input Location

### ISSUE THREAD

**Issue by xiangchenjhu** [ssec-jhu/bluephos#69]
The constants and configuration for ORCA input files were previously temporarily placed in dft_orca.py. It would be better to make these settings user-configurable.

**Comment by xiangchenjhu** [comment]
These ORCA configuration have now been moved to dft_calculators.py in #67 for review.

**Comment by amitschang** [comment]
@xiangchenjhu is this done then? #90 will be treated separately from this so should not block it from closing

**Comment by xiangchenjhu** [comment]
No, currently the dft_calculators.py file includes the settings for ORCA, but they are not configurable from the CLI.
